package/src/components/Message/utils/messageActions.ts

```diff
@@ -77,6 +79,10 @@ export const messageActions = <
     actions.push(editMessage);
   }
 
+  if (ownCapabilities.readEvents && !error && !isThreadMessage) {
```

Comment:
Hmm is `ownCapabilities.readEvents` really the capability we're looking for ? Seems a bit counter intuitive, since as far as I know `readEvents` is mostly for whether we can receive (and dispatch) reads from/to other users, but I might be wrong

Reply:
Yep, it should depend on read events. If the read events are not allowed they won't receive any read events and that would not make these action to do anything.